packages/js/product-editor/src/components/more-menu-dropdown/index.tsx

```diff
@@ -44,7 +51,7 @@ export const MoreMenuDropdown = ( {
 				size: 'compact',
 			} }
 		>
-			{ ( onClose ) => children( onClose ) }
+			{ ( onClose: () => void ) => children( onClose ) }
```

Comment:
```suggestion
			{ ( { onClose }: { onClose: () => void } ) => children( onClose ) }
```
This breaks the more menu dropdown, although not introduced in this PR as far as I am aware.